build/shadowsocks_config.js

```diff
@@ -23,16 +23,8 @@ var __extends = (this && this.__extends) || (function () {
     };
 })();
 Object.defineProperty(exports, "__esModule", { value: true });
-/* tslint:disable */
-var isBrowser = typeof window !== 'undefined';
-var b64Encode = isBrowser ? btoa : require('base-64').encode;
-var b64Decode = isBrowser ? atob : require('base-64').decode;
-var URL = isBrowser ? window.URL : require('url').URL;
-var punycode = isBrowser ? window.punycode : require('punycode');
-if (!punycode) {
-    throw new Error("Could not find punycode. Did you forget to add e.g.\n  <script src=\"bower_components/punycode/punycode.min.js\"></script>?");
-}
-/* tslint:enable */
+var punycode = require('punycode');
+var js_base64_1 = require('js-base64');
```

Comment:
Why are we naming it this way?

Reply:
This is the result of compiling `src/shadowsocks_config.ts`. We should support npm package releases so we can remove `build/` from version control.